Property 'on' does not exist on type 'ListenLiveClient' since 3.4

Since upgrading to 3.4 we receive the following typescript warning: `Property 'on' does not exist on type 'ListenLiveClient'.ts(2339)`. Worked fine with 3.3.x and before.

Code is basically the the [live audio example](https://github.com/deepgram/deepgram-js-sdk?tab=readme-ov-file#live-audio) from the readme, see https://stackblitz.com/edit/vitejs-vite-rxpylf?file=src%2Fmain.ts

<img width="713" alt="Screenshot 2024-07-05 at 3 40 30 PM" src="https://github.com/deepgram/deepgram-js-sdk/assets/536225/b08c328f-a572-4c05-aa58-3b9f0d03a82a">

I am still getting this error in 3.5.

error TS2576: Property 'on' does not exist on type 'ListenLiveClient'. Did you mean to access the static member 'ListenLiveClient.on' instead?

@lukeocodes it's back again, we're getting `error TS2339: Property 'on' does not exist on type 'ListenLiveClient'.` all over again :(

would be great if you could add some basic typescript checks to avoid such issues in the future.

> @lukeocodes it's back again, we're getting `error TS2339: Property 'on' does not exist on type 'ListenLiveClient'.` all over again :(

It would be helpful if you could provide details about your TypeScript version, an example of how you're using it (the code, not a screenshot), details about your OS/environment and anything else that may help.

As for this comment;

> would be great if you could add some basic typescript checks to avoid such issues in the future.

TypeScript is inheriently type safe. If there was an issue here when building with our TS settings, it would fail to build. If there was an issue here with the code, our live `examples/` would also throw errors. My thinking is that this is environment related and we need to be stricter with defining our types.

--- a/package.json
+++ b/package.json
@@ -52,6 +52,7 @@
   },
   "dependencies": {
     "@deepgram/captions": "^1.1.1",
+    "@types/node": "^18.19.39",
     "cross-fetch": "^3.1.5",
     "deepmerge": "^4.3.1",
     "events": "^3.3.0",
@@ -64,7 +65,6 @@
     "@flydotio/dockerfile": "^0.4.10",
     "@types/chai": "^4.3.5",
     "@types/mocha": "^9.1.1",
-    "@types/node": "^18.19.39",
     "@types/sinon": "^17.0.3",
     "@types/ws": "^8.5.10",
     "chai": "^4.3.7",

